Dockerfile.agnos

```diff
@@ -88,13 +88,16 @@ RUN export PATH="/usr/local/pyenv/bin:/usr/local/pyenv/shims:$PATH" && \
     pyenv rehash
 
 # Install openpilot python packages
-RUN mkdir -p /tmp/agnos/uv
-COPY ./userspace/pyproject.toml ./userspace/uv.lock /tmp/agnos/uv
+RUN mkdir -p /tmp/agnos/uv-agnos /tmp/agnos/uv-openpilot
+COPY ./userspace/pyproject.toml ./userspace/uv.lock /tmp/agnos/uv-agnos
+RUN cd /tmp/agnos/uv-openpilot && \
+    curl -sO https://raw.githubusercontent.com/commaai/openpilot/master/pyproject.toml && \
+    curl -sO https://raw.githubusercontent.com/commaai/openpilot/master/uv.lock
```

Comment:
if i build this in 2 years, it's gonna pull the uv.lock from the latest openpilot.

ideally, we can check in a `uv.lock` and refresh that as needed.

Reply:
done